### PR TITLE
Feature gui v2

### LINKED ISSUE #103: GUI upgrade v2

### PR DESCRIPTION
**Pull request by JayChenYJ**
1. Removed status bar.
2. Revamped the colour scheme of StoreMando.
3. Added 1 shortcut key for 'Exit'.
4. Combined 2 menu bar items into 1 called 'Shortcuts'
5. Rearranged css files for better readability.
6. Added welcome message for StoreMando.

7. TODO Make 2 items in one row instead of 1 item. (Possibly look like stickers)
8. TODO Give implicit expiring item warning background colour.

Closes #103

### CODE REVIEW

File: src/main/java/seedu/storemando/logic/parser/ReminderCommandParser.java [hunk 1]
```diff
@@ -12,7 +12,8 @@
 public class ReminderCommandParser implements Parser<ReminderCommand> {
     /**
      * Convert the given number and the time unit to the number of days.
-     * @param num The number use to covert to days
+     *
+     * @param num      The number use to covert to days
```
Comment (kumsssss) [inline src/main/java/seedu/storemando/logic/parser/ReminderCommandParser.java]: I think can change this back.

**Review by kumsssss — COMMENTED** [review]
minor change to be made. Otherwise, LGTM!

File: src/main/java/seedu/storemando/ui/ResultDisplay.java [hunk 2]
```diff
@@ -25,4 +25,8 @@ public void setFeedbackToUser(String feedbackToUser) {
         resultDisplay.setText(feedbackToUser);
     }
 
+    public void welcomeMsg() {
+        resultDisplay.setText("Welcome to StoreMando!!!");
```
Comment (Md-Fazil) [inline src/main/java/seedu/storemando/ui/ResultDisplay.java]: My personal opinion is that the welcome message should just be "Welcome to StoreMando!". Just a suggestion to consider.

**Review by Md-Fazil — COMMENTED** [review]
LGTM!

**Review by kumsssss — APPROVED** [review]
LGTM!

**Pull request merged**